pints/tests/test_log_likelihoods.py

```diff
@@ -1405,6 +1405,150 @@ def simulate(self, x, times):
         self.assertAlmostEqual(log1(0) + log2(0), log3(0))
 
 
+class TestLogNormalLogLikelihood(unittest.TestCase):
+
+    @classmethod
+    def setUpClass(cls):
+        # sreate test single output test model
+        cls.model_single = pints.toy.ConstantModel(1)
+        cls.model_multiple = pints.toy.ConstantModel(2)
+        cls.times = [1, 2, 3, 4]
+        cls.data_single = [3, 4, 5.5, 7.2]
+        cls.data_multiple = [[3, 1.1],
+                             [4, 3.2],
+                             [5.5, 4.5],
+                             [7.2, 10.1]]
+        cls.problem_single = pints.SingleOutputProblem(
+            cls.model_single, cls.times, cls.data_single)
+        cls.problem_multiple = pints.MultiOutputProblem(
+            cls.model_multiple, cls.times, cls.data_multiple)
+        cls.log_likelihood = pints.LogNormalLogLikelihood(cls.problem_single)
+        cls.log_likelihood_adj = pints.LogNormalLogLikelihood(
+            cls.problem_single, mean_adjust=True)
+        cls.log_likelihood_multiple = pints.LogNormalLogLikelihood(
+            cls.problem_multiple)
+        cls.log_likelihood_multiple_adj = pints.LogNormalLogLikelihood(
+            cls.problem_multiple, mean_adjust=True)
+
+    def test_bad_constructor(self):
+        # tests that bad data types result in error
+        data = [0, 4, 5.5, 7.2]
+        problem = pints.SingleOutputProblem(
+            self.model_single, self.times, data)
+        self.assertRaises(ValueError, pints.LogNormalLogLikelihood,
+                          problem)
+
+    def test_call(self):
+        # test calls of log-likelihood
+
+        # single output problem
+        sigma = 1
+        mu = 3.7
+        log_like = self.log_likelihood([mu, sigma])
+        self.assertAlmostEqual(log_like, -10.164703123713256)
```

Comment:
I personally don't think that those tests are particularly meaningful, because those are just numbers I can't really interpret. Does it potentially make more sense to test known edge cases to make sure the likelihood's and sensitivities are correct? 
For example the expressions reduce when soln=1, or sigma=1. Then you could compare manually computer values from the parameters, which are easier to understand, and compare them to the methods.